src/main/java/core/basesyntax/AreaAndFullInfo.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface AreaAndFullInfo {
+    double area();
```

Comment:
bad method name. method names should sound like action
```suggestion
    double calculateArea();
```